.github/workflows/tag-and-build.yml

```diff
@@ -7,35 +7,75 @@ on:
       version:
         description: 'Tag to be used for operator image'
         required: true
-        default: '0.0.0-dev'
+        default: 'v0.0.0-dev'
       replaces:
         description: 'The previous semantic version that this tag replaces.'
         required: true
-        default: '0.0.0-dev'
+        default: 'v0.0.0-dev'
+      mcad-version:
+        description: 'Published version of multi-cluster-app-dispatcher'
+        required: true
+        default: 'v0.0.0-dev'
+      codeflare-sdk-version:
+        description: 'Published version of CodeFlare-SDK'
+        required: true
+        default: 'v0.0.0-dev'
+      instascale-version:
+        description: 'Published version of InstaScale'
+        required: true
+        default: 'v0.0.0-dev'
+      is-stable:
+        description: 'Select if the built image should be tagged as stable'
+        required: true
+        type: boolean
+      branch:
+        description: 'Branch to be used for tag and release'
+        required: true
+        default: 'main'
```

Comment:
ok, I can reuse it.
it would prevent a possible scenario of running workflow from main branch to release on some patch branch, though I guess it is not needed.

Reply:
Done, thanks.